app/services/irs_attempts_api/envelope_encryptor.rb

```diff
@@ -31,11 +32,15 @@ def self.formatted_timestamp(timestamp)
     end
 
     def self.decrypt(encrypted_data:, key:, iv:)
-      cipher = OpenSSL::Cipher.new('aes-128-cbc')
+      cipher = OpenSSL::Cipher.new('aes-256-cbc')
       cipher.decrypt
       cipher.key = key
       cipher.iv = iv
-      decrypted = cipher.update(encrypted_data) + cipher.final
+      decrypted = cipher.update(Base16.decode16(encrypted_data)) + cipher.final
```

Comment:
All of the changes here (and in the IRSAttemptsApiController) need to have their tests updated.

Reply:
I have gone through both sets of tests and made updates where necessary, I believe.